scripts/base/frameworks/intel/main.zeek

```diff
@@ -598,6 +617,9 @@ function remove(item: Item, purge_indicator: bool)
 		return;
 		}
 
+	if ( ! hook purge_item(item) )
+		return;
+
```

Comment:
We also (or only - haven't decided yet) need to hook into `remove_indicator()` below, as that's the event that is published by the manager to remove items on workers

The flow of the events in the intel framework is mind-twisting just much too hard to follow IMO. This definitely needs a few tests, also with a live/test cluster.